network/transport/v2/protocol.go

```diff
@@ -20,30 +20,67 @@ package v2
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"time"
+
+	"github.com/nats-io/nats.go"
+	"github.com/sirupsen/logrus"
+	grpcLib "google.golang.org/grpc"
+
 	"github.com/nuts-foundation/nuts-node/core"
+	"github.com/nuts-foundation/nuts-node/crypto"
+	"github.com/nuts-foundation/nuts-node/events"
 	"github.com/nuts-foundation/nuts-node/network/dag"
 	"github.com/nuts-foundation/nuts-node/network/transport"
 	"github.com/nuts-foundation/nuts-node/network/transport/grpc"
-	grpcLib "google.golang.org/grpc"
+	vdr "github.com/nuts-foundation/nuts-node/vdr/types"
 )
 
 var _ grpc.Protocol = (*protocol)(nil)
 
+// Config specifies config for protocol v2
+type Config struct {
```

Comment:
Config is empty, omit?

Reply:
I could've removed it but it's a nice boilerplate to have as I'm sure the v2 protocol will get it's own configuration at some point